quix-frontend/client/src/react-components/file-explorer/TreeItemMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useRef} from 'react';
+import {Dropdown} from '../../lib/ui/components/dropdown/Dropdown';
+import {useOutsideAlerter} from '../../services/hooks';
+
+interface TreeItemMenuProps {
+  menuOptions: {
+    title: string;
+  }[];
+  isOpen: boolean;
+  setIsOpen(isOpen: boolean): void;
+  onMenuClick(index: number): void;
+}
+
+
+export const TreeItemMenu = ({
+  menuOptions,
+  isOpen,
+  setIsOpen,
+  onMenuClick,
+}: TreeItemMenuProps) => {
+  const iconRef = useRef(null);
+  const menuWrapperRef = useRef(null);
+  useOutsideAlerter([iconRef, menuWrapperRef], () => setIsOpen(false));
+
+  return (
+    <Dropdown
+      OptionsWrapper={({options}: any) =>
```

Comment:
why not `optionsWrapper`?

Reply:
OptionsWrapper's type is `React.ComponentType<InjectedDropdownProps>`
which means it used like:
`<OptionsWrapper .... />`